### PR TITLE
Add section to document how to manually include snowboard assets

### PR DESCRIPTION
**Pull request by mjauvin**

### CODE REVIEW

**Comment by LukeTowers** [comment]
Has the `snowboard.extras.css` asset compiler alias been added to the Winter core yet?

**Comment by mjauvin** [comment]
@LukeTowers https://github.com/wintercms/winter/tree/add-snowboard-extras-css

File: snowboard-introduction.md [hunk 1]
```diff
@@ -57,3 +57,36 @@ Or to include both the JavaScript AJAX and HTML data attribute request functiona
...
+
+```twig
+<script src="{{ [
+	'@snowboard.base',
+	'@snowboard.attr',
+	'@snowboard.request',
+	'@snowboard.extras',
+	[other assets here]
+] | theme }}"></script>
+```
```
Comment (LukeTowers) [inline snowboard-introduction.md:91]:
```suggestion
```twig
<link rel="stylesheet" href="{{ [
    '@snowboard.extras.css',
    [other assets here]
] | theme }}">
```

You can use the asset combiner twig filter like this to include your scripts:

```twig
<script src="{{ [
    '@snowboard.base',
    '@snowboard.attr',
    '@snowboard.request',
    '@snowboard.extras',
] | theme }}"></script>
```
```

File: snowboard-introduction.md [hunk 2]
```diff
@@ -57,3 +57,35 @@ Or to include both the JavaScript AJAX and HTML data attribute request functiona
...
+You can use the asset combiner twig filter like this to include your scripts:
+
+```twig
+<script src="{{ [
+    '@snowboard.base',
+    '@snowboard.attr',
+    '@snowboard.request',
+    '@snowboard.extras',
+] | theme }}"></script>
+``
```
Comment (LukeTowers) [inline snowboard-introduction.md:90]:
```suggestion
```
```

**Comment by LukeTowers** [comment]
@mjauvin can you fix the code block closing backticks manually? GitHub's code suggestion editor can't support edits to the closing sections of code blocks since it confuses them for the ending of the code suggestion block.

**Comment by mjauvin** [comment]
done @LukeTowers

**Pull request merged**